src/Compilers/Core/Portable/InternalUtilities/EnumerableExtensions.cs

```diff
@@ -350,7 +350,7 @@ public static bool Contains<T>(this IEnumerable<T> sequence, Func<T, bool> predi
             return sequence.Any(predicate);
         }
 
-        public static bool Contains(this IEnumerable<string> sequence, string s)
+        public static bool Contains(this IEnumerable<string?> sequence, string? s)
```

Comment:
Do we want multiple versions of this, or perhaps make it generic and `where T : string?`? Otherwise, we'll get a warning if we use this on an enumerable of `string`.

Reply:
Why would we get a warning for `IEnumerable<string>`? The type parameter is covariant so it should convert to `IEnumerable<string?>`

Reply:
Hmm, I hadn't thought about that. Nevermind then.

---
In reply to: [364891203](https://github.com/dotnet/roslyn/pull/40767#discussion_r364891203) [](ancestors = 364891203)